src/Security.php

```diff
@@ -138,8 +138,7 @@ public static function scanFile($file, DOMDocument $dom = null)
     public static function isPhpFpm()
     {
         $isVulnerableVersion = (
-            version_compare(PHP_VERSION, '5.5.22', 'lt')
-            || (
+            (
```

Comment:
We can drop the inner parens as well, as they were used to separate those from the previous conditional. I can do that during merge.